activities.json

```diff
@@ -1018,11 +1018,11 @@
   },
   {
     "ciuName": null,
-    "description": "This document specifies modifications to Fetch which are intended to mitigate the risks associated with unintentional exposure of devices and servers on a client\u2019s internal network to the web at large.",
+    "description": "This document specifies modifications to Fetch which are intended to mitigate the risks associated with unintentional exposure of devices and servers on a client’s internal network to the web at large.",
```

Comment:
Ditto - last one :)

Reply:
Thanks for catching these.  `./activities.py sort` fixed them all up.